ang/civicase/case/directives/case-overview.directive.js

```diff
@@ -156,18 +158,20 @@
     }
 
     /**
+     * Returns the case statuses belonging to the given list of status names.
+     *
+     * The statuses are returned in a lodash wrapped object. This can help
+     * with further manipulations and filterings before extracting the final
+     * value.
+     *
      * @param {string[]} caseStatusNames a list of case status names.
-     * @returns {object[]} the full case status details belonging to the
-     *   given case status names.
+     * @returns {object} A lodash object reference.
      */
-    function getSortedCaseStatusesByName (caseStatusNames) {
+    function getStatusesByName (caseStatusNames) {
```

Comment:
Lets not return `chain` in that case. Because the `getStatusesByName` function should not care about how its return value will be used. So `_.chain` should be used above, where we are calling `getStatusesByName`.

Reply:
Excellent!